### ISSUE TITLE
[1.4.4] syntax error in build.sbt on dumb terminal makes sbt exit immediately

### ISSUE THREAD

**Issue by ritschwumm** [sbt/sbt#6186]
## steps

* add a syntax error to your build.sbt
* start sbt in a dumb terminal (TERM=dumb) like jEdit's

## problem

sbt starts, prints out a message that it cannot load the project, and then immediately exists
```
[info] loading project definition from .../project
.../build.sbt:3: error: not found: value ###
###
^
[error] Type error in expression
[warn] Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore? (default: r)
Process sbt exited with code 1
```

## expectation

sbt should wait for me to enter a choice or let me press enter to retry.
instead, the process just dies.

**Comment by eatkins** [comment]
Yeah, dumb terminals are handled differently from other kinds of terminals. Relevant code is here: https://github.com/sbt/sbt/blob/4d1af7770181813f1c57838dbe7633f8615ada5a/internal/util-logging/src/main/scala/sbt/internal/util/Terminal.scala#L340-L345
